superset/views/core.py

```diff
@@ -1979,6 +1979,8 @@ def dashboard(
         return self.render_template(
             "superset/spa.html",
             entry="spa",
+            # dashboard title is always visible
+            title=dashboard.dashboard_title,
```

Comment:
The test case [`test_get_dashboard`](https://github.com/apache/superset/pull/20703/files#diff-3f2a02719b7e0794608e6f8f1a242ec1cfcebf7e17ebf3dfea7bb664415295ff) actually tests dashboard title in the HTML, but the spa doesn't output it anymore. I'm not sure why current `master` passes the CI. It throws a test error, too, when running locally on my machine.